src/main/resources/schemas/exchange/organizationDirectory.xsd

```diff
@@ -0,0 +1,78 @@
+      <xsd:element name="status" type="xsd:string"/>
+      <xsd:element name="contact" type="tns:contactType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="address" type="tns:addressType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="partOf" type="tns:partOfType" minOccurs="0"/>
+      <xsd:element name="endpoint" type="tns:endpointType" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="contactType">
+    <xsd:sequence>
+      <!--contact use type, for example: techincal, financial -->
+      <xsd:element name="use" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="givenName" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="familyName" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="phone" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="address" type="tns:addressType" minOccurs="0"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="addressType">
+    <xsd:sequence>
+      <!-- Address use type for example, work -->
+      <xsd:element name="use"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="addressLine1"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="addressLine2"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="city"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="state"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="country"  type="xsd:string" minOccurs="0"/>
+      <xsd:element name="zipcode"  type="xsd:string" minOccurs="0"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="endpointType">
+    <xsd:sequence>
+      <!-- Nwhin service name for example, QueryForDoucments -->
+      <xsd:element name="name" type="xsd:string" />
+      <xsd:element name="hcid" type="xsd:string"/>
+      <!--Assigning Authority ID -->
+      <xsd:element name="aaid" type="xsd:string" minOccurs="0"/>
+      <!--service endpoint -->
+      <xsd:element name="serviceURL" type="xsd:anyURI"/>
+      <!--multispec version -->
+      <xsd:element name="serviceVersion" type="xsd:string" />
+      <xsd:element name="payloadType" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="payloadFormat" type="xsd:string" minOccurs="0"/>
+      <!--PurposesOfUse Value Set for SAML header and/or access control-->
+      <xsd:element name="purposeOfUse" type="xsd:string" minOccurs="0"/>
+      <!-- Roles Value Set for SAML header and/or access control-->
+      <xsd:element name="roles" type="xsd:string" minOccurs="0"/>
+      <!--Initiative specific list of use cases supported -->
+      <xsd:element name="usecase" type="xsd:string" minOccurs="0" maxOccurs="unbounded"/>
+    </xsd:sequence>
+  </xsd:complexType>
+  <xsd:complexType name="partOfType">
+    <xsd:annotation>
+      <xsd:documentation>
+        If this is a Carequality Connection, then the partOf object points to the Carequality Implementer.
```

Comment:
The organization partof info seems to be about the exchange.  Should discuss if this should go in the exchange type.  as this might be confusing with the set up we're going with.